Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -501,11 +501,14 @@ final class HTTPClientTests: BaseHTTPClientTests {
         let request = HTTPRequest(method: .get, path: .mockPath)
         let responseData = "{\"message\": \"something is great up in the cloud\"}".asData
         let eTag = "etag"
+        let eTagCreation = Date(timeIntervalSince1970: 1234567)
 
-        self.eTagManager.stubResponseEtag(eTag)
+        self.eTagManager.stubResponseEtag(eTag, creationTime: eTagCreation)
```

Comment:
this isn't etagCreation time anymore it's validation / update time, I'll rename accordingly

Reply:
I can rename this 👍🏻